packages/client/src/index.ts

```diff
@@ -53,6 +53,7 @@ export type PreferenceSettingsType = {
 export type ApiOptions = {
   apiVersion?: string;
   backendUrl?: string;
+  novuUserAgent?: string;
```

Comment:
```suggestion
  UserAgent?: string;
```

Reply:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**question:** a small question, shouldn't we keep it consistent here with camelCasing? for public facing api's? 

I believe it should match the `constructor` signature above?  
```
 constructor({
    apiVersion = DEFAULT_API_VERSION,
    backendUrl = DEFAULT_BACKEND_URL,
    novuUserAgent = NOVU_USER_AGENT,
  }: ApiOptions = {}) {
```

Reply:
Of course. I blame Grammarly for this.